guardian/templates/admin/guardian/model/change_form.html

```diff
@@ -2,7 +2,9 @@
 {% load i18n admin_urls %}
 
 {% block object-tools-items %}
+{% if request.user.is_superuser %}
```

Comment:
I can not accept that way to determine button visibility. It introduces a security risk because the button will have a different display logic from the page to which it directs. I agree that this should be uniform and this is currently an bug. However, we can not solve this in such a simplified way, because we lose flexibility. I rather see in that part reusing that condition: https://github.com/django-guardian/django-guardian/blob/devel/guardian/admin.py#L225
It is possible that this condition should be extracted to separate method, so the result of that method can be overwritten (extra flexibility for end user) and used in two places.